odoo/addons/base/models/ir_actions_report.py

```diff
@@ -395,7 +395,7 @@ def _prepare_html(self, html):
                 res_ids.append(None)
 
         if not bodies:
-            body = b''.join(lxml.html.tostring(c, encoding='utf-8') for c in body_parent.getchildren())
+            body = Markup(''.join(lxml.html.tostring(c, encoding='utf-8') for c in body_parent.getchildren()))
```

Comment:
is there a test for those changes? how come the original change did not break the testsuite?

Reply:
The pdf reports are not tested

Reply:
body = b''.join(lxml.html.tostring(c, encoding='utf-8') for c in body_parent.getchildren())
            body = body.decode('utf-8')